packages/main/src/themes/Label.css

```diff
@@ -37,13 +37,17 @@ bdi {
 }
 
 :host(:not([wrap])[required][show-colon]) .ui5-label-text-wrapper {
-	max-width: calc(100% - .85rem);
+	max-width: calc(100% - .75rem);
 }
 
 :host(:not([wrap])[required]) .ui5-label-text-wrapper {
 	max-width: calc(100% - 0.475rem);
 }
 
+:host([required]:not([wrap])) .ui5-label-text-wrapper {
```

Comment:
The newly added selector looks like the previous one, just the attrs order is switched?

[required]:not([wrap])
vs
:not([wrap])[required]